src/main/java/uk/gov/hmcts/opal/controllers/DraftAccountController.java

```diff
@@ -119,6 +122,25 @@ public ResponseEntity<String> deleteDraftAccountById(
         return buildResponse(String.format(ACCOUNT_DELETED_MESSAGE_FORMAT, draftAccountId));
     }
 
+    @PutMapping(value = "/{draftAccountId}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Replaces an existing Draft Account Entity in the DB with data in request body")
+    public ResponseEntity<DraftAccountResponseDto> replaceDraftAccount(
+        @PathVariable Long draftAccountId,
+        @RequestBody ReplaceDraftAccountRequestDto dto,
+        @RequestHeader(value = "Authorization", required = false) String authHeaderValue) {
+
+        log.info(":PUT:replaceDraftAccount: replacing draft account entity with ID: {} and data: {}",
```

Comment:
We will probably want to remove the dto from the logs in the future, but to help us during development, for now, it's nice to keep it in there

Reply:
Good spot.